components/AddDonation.js

```diff
@@ -0,0 +1,136 @@
+import styled from "styled-components";
+import { useState } from "react";
+import Counter from "./Counter";
+
+export default function AddDonation({ createDonation }) {
+  const [toggleAddButton, setToggleAddButton] = useState(true);
+  const [count, setCount] = useState(0);
+  function handleClick() {
```

Comment:
you can give this function a more descriptive name for example handleToggleForm.

Reply:
changed it :)